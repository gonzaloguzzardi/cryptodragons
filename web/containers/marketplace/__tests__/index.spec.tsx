import React from 'react'
import renderer from 'react-test-renderer'

import Marketplace from '..'

jest.mock('components/app-toolbar', () => ({
  __esModule: true,
  default: 'AppToolbar',
}))
jest.mock('components/footer/desktop', () => ({
  __esModule: true,
  default: 'AppFooter',
}))
jest.mock('components/footer/mobile', () => ({
  __esModule: true,
  default: 'AppFooter',
}))

test('Renders Marketplace desktop', () => {
  const deviceType = 'desktop'
  const component = renderer.create(<Marketplace deviceType={deviceType} />)
  
    expect(component).toMatchSnapshot()
})

test('Renders Marketplace mobile', () => {
  const deviceType = 'mobile'
  const component = renderer.create(<Marketplace deviceType={deviceType} />)
    
    expect(component).toMatchSnapshot()
})
