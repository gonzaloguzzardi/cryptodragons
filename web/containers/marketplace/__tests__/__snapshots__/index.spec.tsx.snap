// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Marketplace desktop 1`] = `
<div
  className="container"
>
  <main>
    <AppToolbar
      accountsState={
        Object {
          "connectToProvider": [Function],
          "mainchain_account": null,
          "provider_installed": false,
          "updateAccountsData": [Function],
        }
      }
      deviceType="desktop"
      section="marketplace"
    />
    <p>
      Marketplace
    </p>
  </main>
</div>
`;

exports[`Renders Marketplace mobile 1`] = `
<div
  className="container"
>
  <main>
    <AppToolbar
      accountsState={
        Object {
          "connectToProvider": [Function],
          "mainchain_account": null,
          "provider_installed": false,
          "updateAccountsData": [Function],
        }
      }
      deviceType="mobile"
      section="marketplace"
    />
    <p>
      Marketplace
    </p>
  </main>
</div>
`;
